packages/venia-ui/lib/components/CategorySort/categorySortItem.js

```diff
@@ -0,0 +1,42 @@
+import React, { useCallback } from 'react';
+import { bool, func, shape, string } from 'prop-types';
+import Icon from '../Icon/icon';
+import { Check } from 'react-feather';
+import { mergeClasses } from '../../classify';
+
+import defaultClasses from './categorySortItem.css';
+
+const CategorySortItem = props => {
+    const { onClick, sortItem, active } = props;
+    const classes = mergeClasses(defaultClasses, props.classes);
+
+    const handleClick = useCallback(() => {
+        onClick(sortItem);
+    }, [sortItem, onClick]);
```

Comment:
Yes, much better. As long as `sortItem` is stable (which it appears to be here), this will be properly memoized: created once and stable across renders.